plugin/flow.vim

```diff
@@ -24,7 +24,13 @@ if !exists("g:flow#qfsize")
   let g:flow#qfsize = 1
 endif
 if !exists("g:flow#flowpath")
-  let g:flow#flowpath = "flow"
+  " Search for a local version of flow
+  let s:npm_local_flowpath = finddir("node_modules", ".;") . "/.bin/flow"
```

Comment:
I think, we need to detect flowpath for every .js file, not only at startup.

Reply:
Yeah good point. Though it'll hurt performance. I implemented it as you suggested.